adapter/internal/api/apis_impl.go

```diff
@@ -121,20 +124,14 @@ func extractAPIProject(payload []byte) (apiProject ProjectAPI, err error) {
 				return apiProject, conversionErr
 			}
 
-			apiType, lifeCycleStatus, productionEndpoint, sandboxEndpoint, endpointImplementationType, err :=
-				extractAPIInformation(apiJsn)
-
-			if endpointImplementationType == "INLINE" {
+			apiProject.APIJsn = apiJsn
+			extractAPIInformation(apiProject)
+			if apiProject.endpointImplementationType == "INLINE" {
```

Comment:
Shall we move "INLINE" to constant?

Reply:
Fixed by https://github.com/wso2/product-microgateway/pull/1747/commits/1b3cf6499331400c8f4545af0f444f99077c971f